### PR TITLE
Updating to RTM builds of xunit and Moq

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb

File: test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/project.json [hunk 1]
```diff
@@ -23,34 +25,20 @@
       "include": "TestFiles/**"
     },
     "compile": {
-      "include": "**/*.cs",
```
Comment (ajaybhargavb) [inline test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/project.json]: :clap:

**Comment by ajaybhargavb** [comment]
:shipit:

**Comment by javiercn** [comment]
:shipit:

**Pull request merged**